webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlSerialisationImpl.scala

```diff
@@ -175,7 +175,7 @@ class ActiveTechniqueSerialisationImpl(xmlVersion:String) extends ActiveTechniqu
   def serialise(activeTechnique:ActiveTechnique):  Elem = {
     createTrimedElem(XML_TAG_ACTIVE_TECHNIQUE, xmlVersion) (
         <id>{activeTechnique.id.value}</id>
-        <techniqueName>{activeTechnique.techniqueName}</techniqueName>
+        <techniqueName>{activeTechnique.techniqueName.value}</techniqueName>
```

Comment:
this one was the likely cause of all the `TechniqueName()` in the technique tree after a reload archive from master